lib/fork.js

```diff
@@ -14,8 +14,8 @@ module.exports = function (args) {
 	var file = args[0];
 
 	var options = {
-		silent: true,
-		cwd: path.dirname(file)
+		cwd: path.dirname(file),
+		stdio: ['ignore', process.stdout, process.stdout]
```

Comment:
I am not sure why we forward both child output streams to `stdout`. I do it here simply because it is what we were doing before. Making that change will impact a lot of tests, and is a separate PR.

In my mind what we are currently doing makes the *least* sense of the the available options. I think we should either:

 1. pipe `child.stdout` -> `process.stdout`, and `child.stderr` -> `process.stderr`.
 2. pipe both to `process.stderr`. This keeps `process.stdout` pristine and will not interfere with our TAP output.